internal/kafka/internal/handlers/validation_test.go

```diff
@@ -499,14 +582,14 @@ func Test_Validation_ValidateKafkaUserFacingUpdateFields(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			gomega.RegisterTestingT(t)
+			RegisterTestingT(t)
 			validateFn := ValidateKafkaUserFacingUpdateFields(tt.arg.ctx, tt.arg.authService, tt.arg.kafka, &tt.arg.kafkaUpdateRequest)
 			err := validateFn()
-			gomega.Expect(tt.want.wantErr).To(gomega.Equal(err != nil))
+			Expect(tt.want.wantErr).To(Equal(err != nil))
 			if !tt.want.wantErr && err != nil {
 				t.Errorf("ValidateKafkaUserFacingUpdateFields() expected not to throw error but threw %v", err)
```

Comment:
This is already checked in the previous `Expect`

Reply:
I didn't introduce this test so I'd rather not make a change here unless you feel strongly about removing this check. Is that ok @ziccardi ?

Reply:
if `tt.want.wantErr` is false and err is not nil,
```go
Expect(tt.want.wantErr).To(Equal(err != nil))
```
will fail, so that test is useless. I believe it adds confusion.

Reply:
Would it make more sense if `Expect(tt.want.wantErr).To(Equal(err != nil))` was included after the if block? I understand that the check would show up unexpected errors which could be useful

Reply:
The detail of the error can be logged with gomega too: https://onsi.github.io/gomega/#annotating-assertions

Reply:
If you prefer to leave it, I'm ok with that: it won't harm. Maybe we could just track that it needs to be cleaned up somewhere and do it later

Reply:
Thanks @ziccardi I'll use annotations to give error details here